app/views/content/blog/jills-story-getting-back-into-the-classroom.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Getting back into the classroom
+date: "2021-01-01"
+author: Jill
+images:
+  jill:
+    path: media/images/content/stories/stories-jill.jpg
+    thumbnail_path: media/images/content/stories/stories-jill.jpg
+    alt: Returning teacher Jill
+description: |-
+  Jill's route into teaching
```

Comment:
Need to have more of a description here as this appears on the blog landing page